pages/static/src/js/groups.js

```diff
@@ -104,7 +143,7 @@ module.exports = function() {
   }
 
   function updateInfo(groups) {
-    var location = '', city, county, state, country;
+    var location = '', city, county, state, country, group_type = null;
```

Comment:
Why need to add `= null`?

Reply:
👍 leftover from different implementation will remove.